vllm/core/scheduler.py

```diff
@@ -449,9 +463,6 @@ def _schedule_running(
                 if curr_loras is not None and seq_group.lora_int_id > 0:
                     curr_loras.add(seq_group.lora_int_id)
 
-        # Make sure all queues are updated.
-        assert len(running_queue) == 0
```

Comment:
is this assert important? sorry I missed this earlier

Reply:
This means len(running_queue) != num_seqs sometimes. I think it is harmless (and it only happens upon preemption). running queue is anyway sorted by FCFS. There's a way to keep this assert, but that will affect the performance a bit